src/bundle/Resources/config/services/normalizers.yaml

```diff
@@ -0,0 +1,24 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    Ibexa\Search\Serializer\Normalizer\Suggestion\ContentSuggestionNormalizer:
+        tags:
+            - { name: 'ibexa.search.serializer.normalizer' }
+
+    Ibexa\Search\Serializer\Normalizer\Suggestion\ParentLocationCollectionNormalizer:
+        tags:
+              - { name: 'ibexa.search.serializer.normalizer' }
+
+    Ibexa\Search\Serializer\Normalizer\Suggestion\ParentLocationNormalizer:
+        tags:
+              - { name: 'ibexa.search.serializer.normalizer' }
+
+    Ibexa\Search\Serializer\SuggestionSerializerFactory: ~
+
+    Ibexa\Search\Serializer\SuggestionSerializer:
+        factory: '@Ibexa\Search\Serializer\SuggestionSerializerFactory'
+        arguments:
+            $normalizers: !tagged_iterator { tag: 'ibexa.search.serializer.normalizer' }
```

Comment:
I expected something more along the lines of:
https://github.com/ibexa/product-catalog/blob/0732ba8a58a197d18c7a518ec6c384bf6417272e/src/bundle/Resources/config/services/serializer.yaml#L9-L13